attributioncode/validator.go

```diff
@@ -101,7 +102,7 @@ func NewValidator(hmacKey string, timeout time.Duration) *Validator {
 }
 
 // Validate validates and sanitizes attribution code and signature
-func (v *Validator) Validate(code, sig string) (*Code, error) {
```

Comment:
it looks like `sig` lost the `string` type - should that be re-added?

Reply:
In golang you can do multiple type declarations at once, so `code`, `sig`, and `refererHeader` are all strings in this case.

Reply:
Ahh, neat!